docs/reference/action_list.rst

```diff
@@ -578,5 +578,55 @@ Checkbox range selection
     You can check / uncheck a range of checkboxes by clicking a first one,
     then a second one with shift + click.
 
+Displaying a non-model field
+----------------------------
+
+The list view can also display fields that are not part of the model class.
+
+In some situations you can add a new getter to your model class to calculate
+a field based on the other fields of your model::
+
+    // src/Entity/User.php
+
+    public function getFullName(): string
+    {
+        return $this->getGivenName().' '.$this->getFamilyName();
+    }
+
+    // src/Admin/UserAdmin.php
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper->addIdentifier('fullName');
+    }
+
+In situations where the data are not available in the model or it is more performant
```

Comment:
I think we should add a `.. versionadded:: 3.x` section here.

Reply:
Thanks for your review Javier.

Reply:
Unfortunately `.. versionadded:: 3.x` is rejected by DOCtor-RST. Any idea how to solve this?

Reply:
Last time, I guessed the version and added something like 3.59 directly...

@OskarStark Can you help us ?

Reply:
I found a solution at https://github.com/OskarStark/doctor-rst/issues/650#issuecomment-606028508.